src/nc_message.c

```diff
@@ -650,6 +655,12 @@ msg_parse(struct context *ctx, struct conn *conn, struct msg *msg)
         status = NC_OK;
         break;
 
+    case MSG_PARSE_ERROR:
```

Comment:
My only concern here is the behavioral change due to the enomem case in memcached message parsing as well as how this change would affect redis clients.